modules/gui/include/visp3/gui/vpPclPointCloudVisualization.h

```diff
@@ -0,0 +1,252 @@
+/****************************************************************************
+ *
+ * ViSP, open source Visual Servoing Platform software.
+ * Copyright (C) 2005 - 2023 by Inria. All rights reserved.
+ *
+ * This software is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ * See the file LICENSE.txt at the root directory of this source
+ * distribution for additional information about the GNU GPL.
+ *
+ * For using ViSP with software that can not be combined with the GNU
+ * GPL, please contact Inria about acquiring a ViSP Professional
+ * Edition License.
+ *
+ * See http://visp.inria.fr for more information.
+ *
+ * This software was developed at:
+ * Inria Rennes - Bretagne Atlantique
+ * Campus Universitaire de Beaulieu
+ * 35042 Rennes Cedex
+ * France
+ *
+ * If you have questions regarding the use of this file, please contact
+ * Inria at visp@inria.fr
+ *
+ * This file is provided AS IS with NO WARRANTY OF ANY KIND, INCLUDING THE
+ * WARRANTY OF DESIGN, MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE.
+ * 
+ * 
+ *
+ * Description:
+ * Real-time 3D point clouds plotter based on the PCL library.
+ *
+ * Authors:
+ * Romain LAGNEAU
+ *
+ *****************************************************************************/
+
+#ifndef POINTCLOUDVISUALIZATION_H
+#define POINTCLOUDVISUALIZATION_H
+
+#ifndef DOXYGEN_SHOULD_SKIP_THIS
+
+#include <visp3/core/vpConfig.h>
+
+#if defined(VISP_HAVE_PCL)
+// System
+#include <thread>
+#include <mutex>
+
+// ViSP
+#include <visp3/core/vpColVector.h>
+
+// PCL
+#include <pcl/visualization/pcl_visualizer.h>
+
+class VISP_EXPORT vpPclPointCloudVisualization
```

Comment:
Class name should be changed since `Pcl` is for PointCloudLibrary that is redundant with `PointCloud`.
I suggest rather `vpPclVisualizer` or `vpPointCloudVisualizer` as you prefer

Reply:
Done in [fdc4d44](https://github.com/lagadic/visp/pull/1163/commits/fdc4d44146c4e0fdd47bb2af90e0bdf765228f88)